core-blocks/video/index.js

```diff
@@ -191,11 +226,11 @@ export const settings = {
 	},
 
 	save( { attributes } ) {
-		const { src, caption, align } = attributes;
+		const { src, caption, align, width, height } = attributes;
 		return (
 
 			<figure className={ align ? `align${ align }` : null }>
-				{ src && <video controls src={ src } /> }
+				{ src && <video controls src={ src } width={ width } height={ height } /> }
```

Comment:
The dimensions are being applied in the save, but not in the edit. This will not allow the user to preview the dimensions in the editor. Should we also set them in the edit method?

Reply:
Thank you for catching this - I'll address it ASAP.

Reply:
Addressed with https://github.com/WordPress/gutenberg/pull/6530/commits/bf92d8cbd47c86dfe578ab72f6ae4a002aac38f9